src/aat/resources/features/F-103/F-103_Test_Case_Access_Base_Dil_C3.td.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_guid_": "F-103_Test_Case_Access_Base_Dil_C3",
+	"method": "PUT",
+	"uri": "/cases/{cid}/users/{uid}",
+	"users": {
+    "invokingUser": {
+      "username": "befta.caseworker.caa.1@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userRichard": {
+      "username": "befta.caseworker.2.solicitor.2@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userDil": {
+      "username": "befta.caseworker.2.solicitor.3@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userSteve": {
+      "username": "befta.caseworker.2.solicitor.4@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    }
```

Comment:
This lines seem to be unnecessarily repeated. Am I missing a detail?

Reply:
JSON-based inheritance mechanism would already bring the same lines from a parent, base object.

Reply:
Done, used extends!!!